src/factories/WithFaceLandmarks.ts

```diff
@@ -5,64 +5,130 @@ import { isWithFaceDetection, WithFaceDetection } from './WithFaceDetection';
 
 export type WithFaceLandmarks<
   TSource extends WithFaceDetection<{}>,
-  TFaceLandmarks extends FaceLandmarks = FaceLandmarks68 > = TSource & {
-    landmarks: TFaceLandmarks,
-    unshiftedLandmarks: TFaceLandmarks,
-    alignedRect: FaceDetection,
-    angle: { roll: number | undefined, pitch: number | undefined, yaw: number | undefined },
-  }
+  TFaceLandmarks extends FaceLandmarks = FaceLandmarks68
+> = TSource & {
+  landmarks: TFaceLandmarks;
+  unshiftedLandmarks: TFaceLandmarks;
+  alignedRect: FaceDetection;
+  angle: {
+    roll: number | undefined;
+    pitch: number | undefined;
+    yaw: number | undefined;
+  };
+};
 
-export function isWithFaceLandmarks(obj: any): obj is WithFaceLandmarks<WithFaceDetection<{}>, FaceLandmarks> {
-  return isWithFaceDetection(obj)
+export function isWithFaceLandmarks(
+  obj: any,
+): obj is WithFaceLandmarks<WithFaceDetection<{}>, FaceLandmarks> {
+  return (
+    isWithFaceDetection(obj)
     // eslint-disable-next-line dot-notation
     && obj['landmarks'] instanceof FaceLandmarks
     // eslint-disable-next-line dot-notation
     && obj['unshiftedLandmarks'] instanceof FaceLandmarks
     // eslint-disable-next-line dot-notation
-    && obj['alignedRect'] instanceof FaceDetection;
+    && obj['alignedRect'] instanceof FaceDetection
+  );
 }
 
 function calculateFaceAngle(mesh) {
-  // returns the angle in the plane (in radians) between the positive x-axis and the ray from (0,0) to the point (x,y)
-  const radians = (a1, a2, b1, b2) => (Math.atan2(b2 - a2, b1 - a1) % Math.PI);
-  // convert radians to degrees
+  /*
```

Comment:
I do not include authoring notes in the code itself. Authors for a specific part of the code are visible via git blame, history and included as contributors on github itself.